templates/min/web3js-angular-dapp-min/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "web3js-angular-dapp-min",
+  "version": "0.1.0",
+  "description": "Minimal template for bootstrapping a Web3.js + Angular dApp",
+  "contributors": [
+    "ChainSafe <info@chainsafe.io>"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
```

Comment:
For this template what do you think for adding `build` (and later `test` ) in CLI tool's github CI, so with every PR we will be able to detect if any thing is breaking.

Reply:
Great idea! I will add that 💯